packages/components/src/menu-item/style.scss

```diff
@@ -41,11 +41,19 @@
 		// !important allows icons from plugins to be overriden and given a dark-gray fill
 		fill: $dark-gray-900 !important;
 	}
+
+	// Don't wrap text.
+	@include break-mobile() {
+		.components-popover:not(.is-mobile) & {
```

Comment:
This is a bit confusing because it's used on mobile breakpoints but when `:not(.is-mobile)` is true. 😕

Reply:
The "break-mobile" mixin is the confusing aspect. `is-mobile` never invokes on desktop responsive viewpoints, but the mixin does as it is just a media query. So in a way this is a _responsive_ fix, not solely a mobile fix. And given the whole behavior of the popover changes when is-mobile is present, I'd rather not interfere here. 

I will clarify the comment.

Reply:
lol, paths crossed! Cheers 👍